engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -420,12 +420,22 @@ public void setRequest(LogicalPlanRequest request)
   {
     private String containerId;
     private String errorMessage;
+    private String logFileName;
```

Comment:
I want to avoid using objects here. If you check FSEventRecorder. it uses BeanUtils to read properties of the Event object as key value pairs in map. If we use objects inside StramEvent, I think the conversion of event to key-value pair map will have to change. So won't use LogInformation object here.